t/re/reg_mesg.t

```diff
@@ -747,6 +747,12 @@ my @experimental_vlb = (
     '/(?| (?=(foo)) | (?<!(foo)|p) )/x'
             => 'Variable length negative lookbehind with capturing' .
                ' is experimental {#} m/(?| (?=(foo)) | (?<!(foo)|p) ){#}/',
+    '/(?<!(foo|bloop(*ACCEPT)|bar)baz)/'
+            => 'Variable length negative lookbehind with capturing' .
+               ' is experimental {#} m/(?<!(foo|bloop(*ACCEPT)|bar)baz){#}/',
+    '/(?<=(foo|bloop(*ACCEPT)|bar)baz)/'
+            => 'Variable length positive lookbehind with capturing' .
+               ' is experimental {#} m/(?<=(foo|bloop(*ACCEPT)|bar)baz){#}/',
```

Comment:
I would have expected this to test something where the ACCEPT was what changed it from fixed to variable length: `/(?<=(ab|c(*ACCEPT)d))/`, and perhaps also the reverse: `/(?<=(ab|cd(*ACCEPT)efg))/` (though if the latter were wrongly to warn I'd have no problem living with it).

Reply:
Ah good suggestion. Ill tweak that and push a fix now for rebase afterwards. (I'm learning. ;-)

Reply:
Pushed.

Reply:
> BTW, I didn't add `/(?<=(ab|cd(*ACCEPT)efg))/` as it does warn as you suspected it might, and I want to add a new issue to make it not warn. But I don't want to burden this PR with that fix as it could be serious work and I'd really like to see this released in 5.36. I just had some ideas how to make this better, but perfect is the enemy of good enough and all that.

Yep, that's fine by me.